airflow/providers/dbt/cloud/operators/dbt.py

```diff
@@ -218,3 +218,55 @@ def execute(self, context: Context) -> None:
                 json.dump(response.json(), file)
             else:
                 file.write(response.text)
+
+
+class DbtCloudListJobsOperator(BaseOperator):
+    """
+    List jobs in a dbt Cloud project.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:DbtCloudListJobsOperator`
+
+    Retrieves metadata for all jobs tied to a specified dbt Cloud account. If a ``project_id`` is
+    supplied, only jobs pertaining to this project id will be retrieved.
+
+    :param account_id: Required. The ID of a dbt Cloud account.
+    :param order_by: Optional. Field to order the result by. Use '-' to indicate reverse order.
+        For example, to use reverse order by the run ID use ``order_by=-id``.
+    :param project_id: Optional. The ID of a dbt Cloud project.
+    """
+
+    template_fields = (
+        "account_id",
+        "project_id",
+    )
+
+    def __init__(
+        self,
+        *,
+        dbt_cloud_conn_id: str = DbtCloudHook.default_conn_name,
+        account_id: int,
+        project_id: int | None = None,
+        order_by: str | None = None,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.dbt_cloud_conn_id = dbt_cloud_conn_id
+        self.account_id = account_id
+        self.project_id = project_id
+        self.order_by = order_by
+        self.hook: DbtCloudHook
+
+    def execute(self, context: Context) -> None:
+        self.hook = DbtCloudHook(self.dbt_cloud_conn_id)
+        list_jobs_response = self.hook.list_jobs(
+            account_id=self.account_id, order_by=self.order_by, project_id=self.project_id
+        )
+        self.log.info(
+            "Jobs in the specified dbt Cloud account %s and project %s are:",
+            str(self.account_id),
+            str(self.project_id),
+        )
+        for job in list_jobs_response:
+            self.log.info(job.json()['data'])
```

Comment:
Should not we return the job id list? This can be stored in an XCom value and something useful can be done with it (for example you coud pass it to a Dynamically mapped task to wait for the jobs to complete and do something with them?

Reply:
Yes makes sense @potiuk .  I will make the changes accordingly.

Reply:
This is addressed